src/app/containers/ImageWithPlaceholder/index.jsx

```diff
@@ -40,23 +42,32 @@ const ImageWithPlaceholder = ({
     <Image onLoad={() => setIsLoaded(true)} {...imageProps} />
   );
 
+  const shouldPreload = !isAmp && preload;
+
   return (
-    <ImagePlaceholder css={isLoaded && `background: none`} ratio={ratio}>
-      {isAmp ? (
-        <AmpImg
-          alt={alt}
-          attribution={copyright || ''}
-          layout="responsive"
-          src={src}
-          srcset={srcset}
-          height={height}
-          width={width}
-        />
-      ) : (
-        renderImage(imageToRender, lazyLoad, fallback)
+    <>
+      {shouldPreload && (
+        <Helmet>
+          <link rel="preload" as="image" href={src} imagesrcset={srcset} />
+        </Helmet>
```

Comment:
I think we should have some tests for:
- image preload tag is not added on amp
- image preload tag is added on canonical when preload is true
- image preload tag is not added_on canonical when preload is false